include/zephyr/bluetooth/mesh/blob_srv.h

```diff
@@ -138,6 +138,7 @@ struct bt_mesh_blob_srv {
 	struct bt_mesh_blob_block block;
 	const struct bt_mesh_model *mod;
 	enum bt_mesh_blob_xfer_phase phase;
+	uint16_t chunk_size;
```

Comment:
Should we add a similar change to the client? (Shouldn't both be configurable at runtime through the later API changes?)

Reply:
Actually, I might need to change the kconfig range for that one, looking at it now. As the client will choose the minimum chunk size that can be supported by all participating servers, @omkar3141 said it shouldn't be set lower than BT_MESH_TX_SDU_MAX - 7 (overhead). 
Not sure now why that was again @omkar3141 ? (because we don't want to give the option to clamp down the chunk size from client side?)

Reply:
Reverted this to draft for now, until this is solved. Probably was a bit fast to change the status friday before going home :)

Reply:
@HaavardRei It is already there for the client :). Inside the `caps` struct, hence I did not mention it before in any of the discussions. See [here](https://github.com/zephyrproject-rtos/zephyr/pull/76766/files#diff-6a5bc1b916a9946dec1cd3d6e6b680f6d8d11ccbd01c14d56e7510a506f337e2R1497)

Reply:
> Not sure now why that was again @omkar3141 ? (because we don't want to give the option to clamp down the chunk size from client side?)

No. We want to give, but not now. @KyraLengfeld Currently such option does not exist, but this newly added KConfig option for client will give compile time control, and the `max_chunk_size` in  `caps` struct, sort of gives run-time control (by commenting out above mentioned line), if someone really want to hack it that way (in the absence of API).

Reply:
I'm not sure if this is required at all. Please take a look at my suggestion.